netanalysis/blocktest/measure.sh

```diff
@@ -24,6 +24,16 @@ declare -ir CURLE_GOT_NOTHING=52  # Injected FIN triggers this.
 declare -ir CURLE_RECV_ERROR=56  # We get this for connection reset by peer.
 declare -ir CURLE_SSL_CACERT=60  # Could be MITM.
 
+function is_online() {
+  # Test signal
+  local response
+  response=$(curl --silent --dump-header - http://www.gstatic.com/generate_204 2> /dev/null)
```

Comment:
include a comment explaining what this url is for

Reply:
Done.